src/mlpack/methods/decision_tree/decision_tree_regressor_impl.hpp

```diff
@@ -165,13 +165,12 @@ DecisionTreeRegressor<FitnessFunction,
     const size_t maximumDepth,
     DimensionSelectionType dimensionSelector,
     const std::enable_if_t<
-        arma::is_arma_type<
-        typename std::remove_reference<
-        WeightsType>::type>::value>*) : splitInfo()
+        arma::is_arma_type<std::remove_reference_t<WeightsType>>::value>*)
```

Comment:
```suggestion
        arma::is_arma_type<std::remove_reference_t<WeightsType>>::value>*) :
```
On my phone I can't do multiline suggestions so this is the first of two parts...

Reply:
Done.